src/components/_templates/Sort/Sort.tsx

```diff
@@ -229,30 +230,28 @@ class Visualiser extends React.Component<Record<string, never>, TVisualiserState
 
   render() {
     return (
-      <div className={VISUALISER.VISUALISER}>
-        <div className={VISUALISER.BUTTONS}>
-          <p className={VISUALISER.TITLE}>{this.state.sorting ? STATUS.SORTING : STATUS.CHOSE_ALGORITHM}</p>
-          <Button classNames={VISUALISER.BUTTON} onClick={this.shuffleItems}>
-            Shuffle
+      <div className={SORT.SORT}>
+        <Menu>
+          <Button href="/" asHref="/" type={BUTTON_TYPE.GREEN}>
+            {"< Back"}
           </Button>
+          <p className={SORT.TITLE}>{this.state.sorting ? STATUS.SORTING : STATUS.CHOSE_ALGORITHM}</p>
```

Comment:
Maybe improve spacing a little :)
```suggestion
          <p className={SORT.TITLE}>
              {this.state.sorting ? STATUS.SORTING : STATUS.CHOSE_ALGORITHM}
          </p>
```

Reply:
Didn't you have prettier warning for this btw?

Reply:
I just have printWidth: 120 and format on save

Reply:
decrease printWidth to 100 and applied this setting